OrbitGl/TracepointTrack.h

```diff
@@ -7,9 +7,9 @@
 
 #include "EventTrack.h"
 
-class TracepointTrack : public EventTrack {
+class TracepointTrack final : public EventTrack {
```

Comment:
Any reason why this is made final? Fine with me, but seems unrelated.

Reply:
First I added that to help with the refactoring. Then I've seen that all the other Track classes had this final and I left it there for completeness reasons. I can remove it if you prefer.

Reply:
I thought I saw other tracks that are not final. But hey, fine with keeping this :)

Reply:
Ah sorry. I should have explained that more clearly. Some tracks are inherited from, notabley TimerTrack and EventTrack. But all the tracks without subclasses had the final tag.

Reply:
Hmm I don't see the final on any Subclass of TimerTrack.

Reply:
Hmmm. Maybe I'm completely mistaken. Let me just remove it.